firebase-segmentation/src/main/java/com/google/firebase/segmentation/FirebaseSegmentation.java

```diff
@@ -36,12 +39,14 @@
   private final FirebaseInstanceId firebaseInstanceId;
   private final CustomInstallationIdCache localCache;
   private final SegmentationServiceClient backendServiceClient;
+  private final Executor executor;
 
   FirebaseSegmentation(FirebaseApp firebaseApp) {
```

Comment:
Could this constructor delegate to the one below?

Reply:
Sure.